index.md

```diff
@@ -134,6 +134,8 @@ If you just want the sheets, you can get them all at my
 (which is, as always, pay-what-you-want).
 They remain on this [GitHub repository](/.src/ECON120) too.
 
+Some [notes](https://grave-space-21c.notion.site/ECON120-c0c1cdb463ca43ad900abb45cbcdf744?pvs=4) by Benny!
```

Comment:
Can you add tracking classes (the `{:.a#course--title--author}` things) to the links? I keep analytics on what resources are being used.